se.cendio.tlclient.json

```diff
@@ -0,0 +1,58 @@
+{
+	"app-id": "se.cendio.tlclient",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "21.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "tlclient-openconf",
+	"tags": [
+		"proprietary"
+	],
+	"finish-args": [
+		"--persist=.fltk",
+		"--persist=.thinlinc",
+		"--share=ipc",
+		"--share=network",
+		"--socket=x11",
+		"--socket=pulseaudio"
+	],
+	"modules": [
+		{
+			"name": "thinlinc",
+			"buildsystem": "simple",
+			"build-commands": [
+				"tar -xf thinlinc.tar.gz",
+				"mv tl-*-client-linux-dynamic-x86_64/* /app",
+				"mv /app/lib/tlclient/share /app",
+				"mv /app/share/applications/thinlinc-client.desktop /app/share/applications/${FLATPAK_ID}.desktop",
+				"mv /app/share/mime/packages/thinlinc-client-mime.xml /app/share/mime/packages/${FLATPAK_ID}.xml",
+				"desktop-file-edit --set-key=TryExec --set-value=/app/bin/tlclient-openconf /app/share/applications/${FLATPAK_ID}.desktop",
+				"desktop-file-edit --set-key=Exec --set-value='/app/bin/tlclient-openconf %f' /app/share/applications/${FLATPAK_ID}.desktop",
+				"desktop-file-edit --set-key=Icon --set-value=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.desktop",
+				"rename thinlinc-client ${FLATPAK_ID} /app/share/icons/hicolor/*/apps/*",
+				"rename application-vnd.cendio.thinlinc.clientconf ${FLATPAK_ID} /app/share/icons/hicolor/*/mimetypes/*",
+				"install -Dm644 ${FLATPAK_ID}.metainfo.xml -t /app/share/metainfo"
```

Comment:
We could probably start including this with the ThinLinc client if that helps.

Reply:
That would be nice. This can also be used by the .deb and .rpm packages. However, note that you'd then ideally also make sure the manifest contains the latest releases in the `<releases>` section.